assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -193,6 +194,49 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
     },
     [wpApi, dataAdapter]
   );
+  const createStoryFromTemplate = useCallback(
+    async (template) => {
+      try {
+        const { createdBy, pages, title, version } = template;
+        const copyTemplateTitle = sprintf(
+          /* translators: %s: template title */
+          __('%s (Copy)', 'web-stories'),
+          title
+        );
+
+        const storyPropsToSave = await getStoryPropsToSave({
+          story: {
+            title: copyTemplateTitle,
+            status: 'draft',
```

Comment:
```suggestion
            status: 'auto-draft',
```

@BrittanyIRL can you try this? (need to merge in the master branch first)

Reply:
this is great! thank you!